pkg/agent/openflow/pipeline.go

```diff
@@ -2106,8 +2063,8 @@ func (f *featureNetworkPolicy) dnsPacketInFlow(conjunctionID uint32) binding.Flo
 	return AntreaPolicyIngressRuleTable.ofTable.BuildFlow(priorityDNSIntercept).
 		Cookie(f.cookieAllocator.Request(f.category).Raw()).
 		MatchConjID(conjunctionID).
-		Action().LoadToRegField(CustomReasonField, CustomReasonDNS).
-		Action().SendToController(uint8(PacketInReasonNP)).
+		Action().SendToController([]byte{uint8(PacketInCategoryDNS)}, true).
```

Comment:
Add some comments to explain why resume should be "true"?

Reply:
Done.